lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -248,7 +264,8 @@ def test_empty_query(
             expected_status = HTTPStatus.FORBIDDEN
 
         index = self.build_index(
-            server_config, self.date_range(self.payload["start"], self.payload["end"])
+            server_config,
+            dates=self.date_range(self.payload.get("start"), self.payload.get("end")),
         )
```

Comment:
If you're going to use a keyword for one of the arguments here, you might as well do it for both.  However, I would just get rid of the parameter definition that you're not using (or, at least, move it to the end of the signature) and then omit the keywords.